test/integration/targets/prepare_http_tests/tasks/default.yml

```diff
@@ -49,11 +49,13 @@
   when: ansible_os_family == 'FreeBSD'
 
 - name: MacOS - Retrieve test cacert
-  get_url:
-    url: "http://ansible.http.tests/cacert.pem"
-    dest: "/usr/local/etc/openssl/certs/ansible.pem"
-  when: ansible_os_family == 'Darwin'
-
-- name: MacOS - Update ca certificates
-  command: /usr/local/opt/openssl/bin/c_rehash
   when: ansible_os_family == 'Darwin'
+  block:
+    - uri:
+        url: "http://ansible.http.tests/cacert.pem"
+        return_content: true
+      register: cacert_pem
+
+    - blockinfile:
+        path: /Library/Frameworks/Python.framework/Versions/3.8/etc/openssl/cert.pem
```

Comment:
This looks like it should use the running Python version instead of being set to `3.8`.

Reply:
Updated to be fully dynamic